packages/xchain-bitcoincash/src/const.ts

```
@@ -76,6 +79,19 @@ export const HaskoinDataProviders: UtxoOnlineDataProviders = {
   [Network.Mainnet]: mainnetHaskoinProvider,
 }
 
+// Blockbook data providers for Bitcoin Cash
+const mainnetBchBlockbookProvider = new BlockbookProvider(
+  'https://bch.nativeswap.io/api/v2',
+  AssetBCH,
+  BCH_DECIMAL,
```

ESLint

[error] Replace `⏎··'https://bch.nativeswap.io/api/v2',⏎··AssetBCH,⏎··BCH_DECIMAL,` with `'https://bch.nativeswap.io/api/v2',·AssetBCH,·BCH_DECIMAL,·{`

(prettier/prettier)